velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2858,6 +2863,57 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+// The test for the alias now() is integrated into this test.
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
+  const auto evaluateCurrentTimestamp = [&](const std::string& functionName,
+                                            int64_t expectedTimestamp,
+                                            const std::string& timeZone) {
+    auto emptyRowVector = makeRowVector(ROW({}), 1);
+
+    core::TypedExprPtr callExpr = std::make_shared<const core::CallTypedExpr>(
```

Comment:
You can use auto here. The type CallTypedExpr can be inferred from the make_shared template type.